Project.toml

```diff
@@ -71,19 +87,12 @@ RelocatableFolders = "0.3, 1"
 Requires = "1"
 Scratch = "1"
 Showoff = "0.3.1, 1"
-PrecompileTools = "1"
 StatsBase = "0.33"
 UnicodeFun = "0.4"
 UnicodePlots = "3.4"
+UnitfulLatexify = "1"
 Unzip = "0.1 - 0.2"
-julia = "1.6"
-
-[extensions]
-FileIOExt = "FileIO"
-GeometryBasicsExt = "GeometryBasics"
-IJuliaExt = "IJulia"
-ImageInTerminalExt = "ImageInTerminal"
-UnitfulExt = "Unitful"
+julia = "1.9"
```

Comment:
Why are we dropping support for LTS ?

Reply:
I don't know exactly how to make tests pass on earlier versions. The failures can be seen in https://github.com/JuliaPlots/Plots.jl/actions/runs/4993428661

Reply:
Since `LaTeXStrings` is a hard  `Plots` dependency, you shouldn't use `@ext_imp_use` but regular imports or `Plots.LaTeXStrings...`.

Also, I'd sugest to add the `UnitfulLatexify` dependency to the `[weakdeps]` and `[extensions]` sections since it is only used in a extension.

Reply:
I thought we were aiming at reducing `Plots` dependencies.
Since `Unitful` is a weak dependency, I think `UnitfulLatexify` should also be an extension.
I'm sure you can make it work through extensions with a bit of effort.

Reply:
https://julialang.slack.com/archives/C67EFTEF3/p1684332705496899